examples/tacopie/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2016, Ruslan Baratov
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-tacopie)
+
+# download tacopie
+hunter_add_package(tacopie)
+
+# now tacopie can be used
+find_package(tacopie CONFIG REQUIRED)
```

Comment:
Thanks. I read the docs, but I still don't know how to apply it to this case. Don't I need to have a forked tacopie repo to install the tacopieConfig.cmake file?

How can I achieve this using only the hunter.cmake file in the hunter repo?

Sorry if this is a silly question. I'm still quite new to this.